cms/.env.example

```diff
@@ -5,6 +5,8 @@ VITE_CLIENT_APP_URL="http://localhost:4174"
 VITE_CLIENT_IMAGES_URL="http://localhost:9000/images"
 VITE_LOGO="../src/assets/logo.svg"
 
+VITE_FAV_ICON="/favicon.png"
```

Comment:
Can we rename this to `VITE_LOGO_FAVICON`? Then it aligns with the logo naming. Perhaps also move the favicon to the `/src/assets/ directory`.